File: C++/palindrome number.php
<?php
$n=1221;
$oi=$n;
$ri=0;

while($n!=0)
{
$r=$n%10;
$ri= $ri*10+ $r;
$n=(int)($n/10);
}
if($oi==$ri)
{
echo "$oi is a palindrome number";
}
else 
{
echo "$oi is not a palindrome number";
}
?>

File: C++/palindromenumber.cpp
#include<iostream>
using namespace std;
int main()
{
int n,rev=0,temp;
cin>>n;            //spose, n=12
    temp=n;
while(temp!=0)
    {
rev=rev*10;      //rev= 0*10 = 0
rev=rev+temp%10; //rev= 0+12%10 =1.2
    temp=temp/10;//temp= 12/10=1.2
}
    if(n==rev)
{
cout<<n<<" is a palindrome number";
}
    else
    {
cout<<n<<" is not a palindrome number";
}
}